minibot.py

```diff
@@ -0,0 +1,260 @@
+"""Contains modifications and extensions to RobotPy
+ needed for the minibot project"""
+
+import wpilib
+import networktables
+
+import pwmoutput
+import micro_maestro_interface
+
+
+class Minibot(wpilib.SampleRobot):
+    """Subclass of wpilib.SampleRobot, handles Pololu board, etc."""
+
+    def __init__(self, *args, **kwargs):
+        super(Minibot, self).__init__(*args, **kwargs)
+
+        self.ds = DriverStation()
+
+        try:
+            micro_maestro = micro_maestro_interface.MicroMaestro("test", 0)
+        except RuntimeError:
+            # No micro maestro on correct port
+            micro_maestro = None
+
+        self.pwm_watch = pwmoutput.PwmWatch(micro_maestro)
+
```

Comment:
Is the __init__ function the only departure from the SampleRobot class? Does the instance make use of `self.pwm_watch` or `self.ds` anywhere? If not, you could make the contents of this (except for the super call) as an ordinary function called in `robotInit`, right?

Reply:
The instance doesn't need access to `self.ds` anywhere before `robotInit` is called, and I was actually going to remove the reference to `self.pwm_watch` since it's a daemon I can just create it and let it do its thing. So if it would be better we can certainly move that into `robotInit`. The reason I have it in `__init__` right now is I wanted to make the minibots simpler to work with by moving anything that is not standard robotpy into a Minibot class we could subclass. The other robot super-classes don't require a call to `super.robotInit()` so I put it into the `__init__` to keep it consistent.